Support cesium

jacmendt opened the pull request:
The pull-request contains the Support of [Cesium](https://cesium.com/) as Terrain Tile Provider. The usage from [Cesium](https://cesium.com/) as Terrain Tile Provider is possible via the extension settings `global.terrain` and is documented in the README.md. Because of different initialization strategies [Cesium](https://cesium.com/) is configured as follows:

> { "type": "cesium", "token": "some_token", "attribution": "<a href='https://www.maptiler.com/maps/3d' target='_blank'>© MapTiler</a>", "asset": "1" }

In comparison [MapTiler](https://www.maptiler.com) is configured as follows:

> { "type": "maptiler", "attribution": "<a href='https://www.maptiler.com/maps/3d' target='_blank'>© MapTiler</a>", "url": "https://api.maptiler.com/tiles/terrain-quantized-mesh/?key=some_token" }

For more information have a look into the README.md

Further the extension now also serves the CESIUM JS dependencies. Also does the pull-request contains some minor improvements in the README.md.

jacmendt committed:
Update readme and add support for cesium terrain tiles

jacmendt committed:
Add Cesium bundle to extension

albig reviewed (commented):
That's rather huge because of the (minified) version von Cesium in `Resources/Public/Contrib`.

I found no licence information about Cesium here. Could you please add an `Resources/Public/Contrib/Cesium/LICENSE` file? Other minified JavaScript usually contains licence information in the head.

jacmendt committed:
Add license for cesium

jacmendt commented:
@albig i added the [license.md](https://github.com/CesiumGS/cesium/blob/main/LICENSE.md). 

Basically, I added the Cesium library to the project to prevent third-party requests, which from the perspective of GDPR requirements, is the most favorable solution. This of course inflates the project history. If you see a more favorable way here, we're happy to remove this dependency as well.

Another issue is the bundle files in the build directory. These have to be built for the extension logic, but also pollute the git history over time. If you have more favorable solutions here, we can discuss them as well.
